### PR TITLE
Add fabric bot config tool

### PR DESCRIPTION
**Pull request by AlexGhiondea**
/cc @weshaggard @lilyjma @maggiepint

### CODE REVIEW

File: tools/CreateRuleFabricBot/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+This tool allows the creation and management of the fabric bot tool.
```
Comment (weshaggard) [inline tools/CreateRuleFabricBot/README.md]: What is the fabric bot?

**Comment by weshaggard** [comment]
You will need to create a pipeline for it if you want to build, sign, and publish it.

File: tools/CreateRuleFabricBot/README.md (hunk 1 above)
Comment (AlexGhiondea) [reply tools/CreateRuleFabricBot/README.md]: It is the bot that handles, among other things, the service triage. It responds to the 'service attention' + 'service' labels and mentions the right people.

**Comment by AlexGhiondea** [comment]
Is there a way to publish a tool that is not a nuget package somewhere? 

Are there any docs for how to create the pipeline?

**Comment by weshaggard** [comment]
>Is there a way to publish a tool that is not a nuget package somewhere?

We don't have any great solutions for that but it isn't required either. You can build it manually if that is all you need. We could publish it as a zip to a github release if that is helpful.

>Are there any docs for how to create the pipeline?

No. This repo is used as a testing bed so there are multiple different ways of doing pipelines in this repo (I should probably fix that). If you need a pipeline I'd suggest copying https://github.com/Azure/azure-sdk-tools/blob/master/tools/pipeline-generator/ci.yml. It is also a tool and uses dotnet install to setup and run.

**Pull request merged**